### DISCUSSION TITLE
power level adjustment

### DISCUSSION THREAD

**Discussion by angwoodworks** [arkypita/LaserGRBL#875] [Q&A]
i have a 3000mw engraver from universalengravers.com the 300mw blue laser model, the software support through them is absolute garbage, switched over to lasergrbl today, cannot get images to actually burn onto the surface for engraving wood. is there a power level adjustment available.

i was previously using engraver master but it honestly blows, only thing i liked about it is that when i finally got it to do one thing it actually burned dark. 

help please

**Comment by gmmanonymus111** [comment]
You can set laser power during image import. Click open (the icon under connect), then select your image.

- For JPG/PNG images, modify setting to your likings, then click next, there should be the laser power adjustment dialog, along with images size settings.
![imageinport](https://user-images.githubusercontent.com/9658243/74082559-78f4a600-4a5b-11ea-8c57-64d53c4d8aa6.png)
- For SVG, you can set laser power after importing your file.
![svgimport](https://user-images.githubusercontent.com/9658243/74082560-798d3c80-4a5b-11ea-8cf6-737c27caa8b6.png)

**Comment by Schwarzi07** [comment]
You can also lower the maximum spindle speed in the GRBL configuration
